erpnext/hr/doctype/leave_policy/leave_policy.py

```diff
@@ -11,8 +11,18 @@
 
 class LeavePolicy(Document):
 	def validate(self):
+		self.validate_leave_type()
+		
 		if self.leave_policy_details:
 			for lp_detail in self.leave_policy_details:
+				if lp_detail.leave_type in leaves:
+					frappe.throw(_("Leave Policy {} is selected in row {}".format(lp_detail.leave_type)))
```

Comment:
```suggestion
					frappe.throw(_("Leave Policy {} is selected in row {}").format(lp_detail.leave_type))
```

Translations require that string formatting is applied after translation function call.